src/index.html

```diff
@@ -103,6 +103,12 @@
                 <li class="nav-item">
                     <a class="nav-link intrapage-nav" href="#contact">Contact</a>
                 </li>
+
+            </li>
+            <li class="nav-item">
+                <a class="nav-link intrapage-nav" href="#board-members">Board Members</a>
```

Comment:
hmm I'm not sure if this section needs to be added to the nav bar, @alevi22 thoughts?

Reply:
I'm actually leaning toward taking it out of the nav since it's right below the contact section.